### PR TITLE
HTML Form

### PR DESCRIPTION
**Pull request by MTS13**

### CODE REVIEW

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
task isn't ready for review, check your code with [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), and try to pass tests. if you have any questions feel free to ask in Slack

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://MTS13.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://MTS13.github.io/layout_html-form/report/html_report/)
```
Comment (DarkMistyRoom) [inline readme.md:4]: You should add these links to the description https://www.loom.com/share/da672f3d083c4247960d02d43f7d527a

File: src/style.css [hunk 2]
```diff
@@ -1 +1,26 @@
-/* styles go here */
+* {
+  box-sizing: border-box;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:3]: you don't need it here

File: src/style.css [hunk 3]
```diff
@@ -1 +1,26 @@
...
+
+#CARS {
+  display: inline-block;
+  margin-bottom: 10px;
+}
+
+#comment {
+  display: inline-block;
+  margin-bottom: 10px;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:18]: use one class instead all these ids

File: src/style.css [hunk 4]
```diff
@@ -1 +1,26 @@
...
+
+input:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+fieldset:not(:last-child) {
+  margin-bottom: 20px;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:26]: don't stylize tags. use classes

File: src/style.css [hunk 5]
```diff
@@ -1 +1,26 @@
-/* styles go here */
+* {
+  box-sizing: border-box;
+}
+
+#loveCats {
+  display: inline-block;
+  margin-bottom: 12px;
```
Comment (DarkMistyRoom) [inline src/style.css:7]: strange value. distance between fields always has to be 10px

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,184 @@
...
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="get"
+    >
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <label for="sname">
```
Comment (DarkMistyRoom) [inline src/index.html:22]: Add empty lines between multiline sibling blocks of HTML

File: src/index.html [hunk 7]
```diff
@@ -11,7 +11,184 @@
...
+        <br>
+        <label for="lname">
+          Name: 
+        </label>
+        <input 
+          type="text"
+          autocomplete="off"
+          id="lname"
+        >
+        <br>
```
Comment (DarkMistyRoom) [inline src/index.html:39]: don't use `br `here. better use `display: block;`

File: src/index.html [hunk 8]
```diff
@@ -11,7 +11,184 @@
...
+        </select>
+      </fieldset>
+      <label for="btn">
+        <input 
+          type="submit" 
+          value="Submit"
+        >
+      </label>
+      
+    </form>
```
Comment (DarkMistyRoom) [inline src/index.html:191]:
```suggestion
      </label>
    </form>
```

File: src/index.html [hunk 9]
```diff
@@ -11,7 +11,184 @@
...
+        >
+      </fieldset>
+      <fieldset>
+        <legend>
+          Additional info:
+        </legend>
+        <label for="comment" id="comment">
+          Comments:
+        </label>
+        <textarea cols="20" rows="2">
```
Comment (DarkMistyRoom) [inline src/index.html:166]: Every field should have name attribute

Comment (DarkMistyRoom) [reply src/index.html:166]: check it everywhere

Comment (DarkMistyRoom) [inline src/index.html:166]: and where is textarea's id?

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
something goes wrong
image

check all notes and [checklist ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)for this task again

**Pull request closed without merging**